qa/zcash/smoke_tests.py

```diff
@@ -22,7 +22,7 @@
 from slickrpc import Proxy
 from slickrpc.exc import RpcException
 
-DEFAULT_FEE = Decimal('0.00001')
+LEGACY_DEFAULT_FEE = Decimal('0.00001')
```

Comment:
The changes to this file might collide with #6616. Non-blocking, it just means the next merge of `version-5.5.0` to master would have conflicts which need resolving.